projectile.el

```diff
@@ -1358,11 +1358,13 @@ IGNORED-DIRECTORIES may optionally be provided."
 (defun projectile-dir-files-alien (directory)
   "Get the files for DIRECTORY using external tools."
   (let ((vcs (projectile-project-vcs directory)))
-    (cond
+    (-map
+     (lambda (x) (string-remove-prefix "./" x))
```

Comment:
Such a check would be extremely expensive on a big project, therefore I'd recommend to solve this at a different level (e.g. the command invocation itself)

Reply:
Agree. At first, I thought that, since you have at least 8 commands which can be used, doing it here would be safe to make sure all of them are consistent. However, given the comment above the `find` command, it seems that care to make sure they were consistent was taken, and `fd` not following the pattern was just a slip.